src/DataCollector/MercureDataCollector.php

```diff
@@ -16,18 +16,19 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+use Symfony\Component\Mercure\Debug\TraceableHub;
 use Symfony\Component\Mercure\Debug\TraceablePublisher;
 
 final class MercureDataCollector extends DataCollector
 {
-    private $publishers;
+    private $hubs;
 
     /**
-     * @var TraceablePublisher[]
+     * @var TraceablePublisher[]|TraceableHub[]
      */
-    public function __construct(iterable $publishers)
+    public function __construct(iterable $hubs)
```

Comment:
Shouldn't we pass an instance of `HubRegistry` here (and maybe `HubRegistry` could extend `ArrayObject` then)?

Reply:
we also need traceable publishers ( BC reasons ), so i think we should keep it as is until we drop traceable publishers.

Reply:
not sure about `\ArrayObject` also, it's only pure syntactic sugar IMHO :)